### PR TITLE
Parse Epoch time

### PR DESCRIPTION
**Pull request by sferik**
The [RSS 2.0 specification](http://cyber.harvard.edu/rss/rss.html) states:

> All date-times in RSS conform to the Date and Time Specification of RFC 822, with the exception that the year may be expressed with two characters or four characters (four preferred).

However, in practice, various other date-time formats are used in RSS feeds. This fact is already acknowledged by Feedjira, which provides custom date-time parsers for Spanish and Japanese month names.

I have come across several RSS feeds in the wild that use the [Epoch time](https://en.wikipedia.org/wiki/Unix_time) format. For example, this one, which Feedjira was unable to parse:
* http://feeds2.feedburner.com/podcast_rds

This patch adds a `DateTimeEpochParser` within the `DateTimeUtilities` module that parses Epoch time correctly. I’ve added a test to ensure that this functionality will not break in the future.

### CODE REVIEW

File: lib/feedjira/date_time_utilities/date_time_epoch_parser.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# rubocop:disable Style/Documentation
+# rubocop:disable Style/DocumentationMethod
+module Feedjira
+  module DateTimeUtilities
+    class DateTimeEpochParser
+      def self.parse(string)
+        epoch_time = string.to_i
+        return Time.at(epoch_time).to_datetime if epoch_time.to_s == string
+        raise "#{string} is not a valid epoch time"
```
Comment (mikeastock) [inline lib/feedjira/date_time_utilities/date_time_epoch_parser.rb]: Should we raise a custom Feedjira error here?

Comment (sferik) [inline lib/feedjira/date_time_utilities/date_time_epoch_parser.rb]: I’m following the lead of [`DateTimeLanguageParser`](https://github.com/feedjira/feedjira/blob/master/lib/feedjira/date_time_utilities/date_time_language_parser.rb) and [`DateTimePatternParser`](https://github.com/feedjira/feedjira/blob/master/lib/feedjira/date_time_utilities/date_time_pattern_parser.rb).

Comment (mikeastock) [inline lib/feedjira/date_time_utilities/date_time_epoch_parser.rb]: Ahh ok

Comment (sferik) [inline lib/feedjira/date_time_utilities/date_time_epoch_parser.rb]: If I was going to refactor this code, I’d try to make it not use exceptions for control flow. I think that would be worth doing, but I prefer to keep my refactorings separate from my new features.

**Pull request merged**